src/main.rs

```diff
@@ -16,18 +16,19 @@ use cargo_toml::TomlManifest;
 use crossgen::Cli;
 use exitfailure::ExitFailure;
 use std::fs::read;
+use std::path::PathBuf;
 use structopt::StructOpt;
 
 fn main() -> Result<(), ExitFailure> {
   setup_panic!();
   let args = Cli::from_args();
   args.log(env!("CARGO_PKG_NAME"))?;
 
-  let dir = args.dir()?;
+  let dir: PathBuf = args.dir()?;
```

Comment:
Is this line necessary?

Reply:
Most likely not. I thought I was doing type conversion from string to pathbuf, because I looked at the wrong property on `cli.rs`. I'll remove this.